pyV2DL3/eventdisplay/fillEVENTS.py

```diff
@@ -168,6 +168,15 @@ def __fillEVENTS__(edFileIO, select=None):
 
         avPedvar = runSummary["pedvarsOn"][0]
 
+        wobble = np.array([
+            runSummary["WobbleNorth"][0],
+            runSummary["WobbleNorth"][1],
+            runSummary["WobbleWest"][0],
+            runSummary["WobbleWest"][1],
+        ])
+        print(np.sort(wobble))
```

Comment:
Why do you sort the wobble offset values? Your arrays includes N and W wobbles - sorting them would remove mix both wobble directions?